XCUITests/SaveLoginsTests.swift

```diff
@@ -24,6 +24,22 @@ class SaveLoginTest: BaseTestCase {
         navigator.goto(LoginsSettings)
         waitForExistence(app.tables["Login List"])
     }
+    
+    func testLoginsListFromBrowserTabMenu() {
```

Comment:
The test looks great, I have been thinking if it would be worthy adding another test to access the logins list when there is passcode set as in [this test](https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/AuthenticationTest.swift#L130)...wdyt?